shipa/post_install.md

```diff
@@ -0,0 +1,54 @@
+# Shipa
+## Getting Started
+To access the Shipa UI with this installation, you will need to install the Shipa CLI on your local machine.
+
+```
+curl -s https://storage.googleapis.com/shipa-client/install.sh | bash
+```
+
+### Wiring
+Shipa for itself installs Nginx as a default ingress controller for the control plane. If
+leveraging Civo, make sure as of Shipa 1.5.2, to unselect Nginx from the ingrress controllers
```

Comment:
ingress spellings, also civo kubernetes comes with Traefik and not nginx by default so which one a user needs ? the default traefik one would work or they need to deselect that as well ?

Reply:
It's just a heads up, the Shipa Control Plane can't be installed as of yet with an existing Nginx installation as Nginx is the ingress controller between the Control Plane and Shipa Nodes. Having Traefik should be ok I would need to test it. Also what's misspelled (spelling and I are not friends ha)

Reply:
Oh I see it

Reply:
So with testing, doesn't look like Civo can support two external IP addresses, or at least I couldn't figure out how to have more than one external IP with spinning up just a Civo Cluster. WIll advise unselecting Traefik for now. 
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/42775804/151385518-6482e85a-f103-4c1c-b42c-a5cc95b09372.png">